src/app/error.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+
+import { Failed } from '@/assets';
+import CButton from '@/components/CButton';
+
+export default function Error() {
+  const router = useRouter();
+
+  const handleGoBack = () => {
+    router.back();
```

Comment:
Going back is not necessarily the best option. Especially if an API error is thrown. Redirect the user to `/` or `/signin` (it wouldn't matter if they go to siginin for example, because if they have the token they will go to dashboard instead)